src/backend/base/langflow/services/auth/utils.py

```
@@ -473,3 +501,42 @@ def decrypt_api_key(encrypted_api_key: str, settings_service: SettingsService):
+    if token and not token.startswith("Bearer "):
+        token = token.strip()
+
+    if settings.auth_settings.CLERK_AUTH_ENABLED:
+        # Prefer Authorization header; if not available, fallback to Clerk cookies
+        if not token:
+            token = request.cookies.get("__session") or request.cookies.get("access_token_lf")
+
+        if token:
+            try:
+                return await get_current_user_by_jwt(token, db)
+            except HTTPException as e:
```

Ruff (0.11.9)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)